pkg/event/manager.go

```diff
@@ -74,7 +78,7 @@ func (em *eventManager) HandleEvent(event *api.Event) error {
 	err := createWorkerForEvent(event)
 	if err != nil {
 		if cloud.IsAllCloudsBusyErr(err) && event.Retry < maxRetry {
-			log.Info("All system worker are busy, wait for 10 seconds")
+			log.Info("All system worker are busy, wait for 10 seconds, error:%v", err)
```

Comment:
it's can be pending?

Reply:
I agree, i think the interval of retry time is a little short.
and "wait for 10 seconds" does not work according to the code. 
i will track it with https://github.com/caicloud/cyclone/issues/528